### PR TITLE
Add source code for the ZIS Admin UI tutorial part 1

### PR DESCRIPTION
**Pull request by taoza**
The source code is to complement the following tutorial: https://developer.zendesk.com/documentation/integration-services/zis-tutorials/zendesk-app-admin-interface/app-oauth/

### CODE REVIEW

File: v2/zis/admin_ui_tutorial_1/src/translations/en.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "app": {
```
Comment (zach-anthony) [inline v2/zis/admin_ui_tutorial_1/src/translations/en.json:2]: It's probably worth updating these to be somewhat aligned to the actual app description and name

**Review by zach-anthony — APPROVED** [review]
LGTM, I would just update the en.json file

File: v2/zis/admin_ui_tutorial_1/src/translations/en.json (hunk 1 above)
Comment (taoza) [reply v2/zis/admin_ui_tutorial_1/src/translations/en.json:2]: @zach-anthony 💇 https://github.com/zendesk/demo_apps/pull/143/commits/ee75d86bce8e0c66465ced5bbf5f98c08ce8c432

**Review by zendesk-grahamgoh — APPROVED** [review]
Nice use of fixup!

**Pull request merged**